packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -94,6 +94,61 @@ class TimelineSummary {
   int countRasterizations() => _extractGpuRasterizerDrawDurations().length;
 
   /// Encodes this summary as JSON.
+  ///
+  /// Data ends with "_time_millis" means time in milliseconds and numbers in
+  /// the "frame_build_times", "frame_rasterizer_times", "frame_begin_times" and
+  /// "frame_rasterizer_begin_times" lists are in microseconds.
+  ///
+  /// * "average_frame_build_time_millis": Average amount of time spent per
+  ///   frame in the framework building widgets, updating layout, painting and
+  ///   compositing.
+  ///   See [computeAverageFrameBuildTimeMillis].
+  /// * "90th_percentile_frame_build_time_millis" and
+  ///   "99th_percentile_frame_build_time_millis": The p-th percentile frame
+  ///   rasterization time in milliseconds. 90 and 99-th percentile number is
+  ///   usually a better metric to estimate worse cases. See discussion in
+  ///   https://github.com/flutter/flutter/pull/19121#issuecomment-419520765
+  ///   See [computePercentileFrameBuildTimeMillis].
+  /// * "worst_frame_build_time_millis": The longest frame build time.
+  ///   See [computeWorstFrameBuildTimeMillis].
+  /// * "missed_frame_build_budget_count': The number of frames that missed
+  ///   the [kBuildBudget] and therefore are in the danger of missing frames.
+  ///   See [computeMissedFrameBuildBudgetCount].
+  /// * "average_frame_rasterizer_time_millis": Average amount of time spent
+  ///   per frame in the GPU rasterizer.
```

Comment:
We just renamed "GPU thread" to "raster thread" (https://github.com/flutter/flutter/issues/29443), so maybe change "the GPU rasterizer" to "the engine rasterizer"

Reply:
This change involves `kRasterizeFrameEventName`, which is the part of the JSON item name, and part of engine interface. Are we sure we should change it here?

Reply:
Ah, I see that we forgot to rename `GPURasterizer::Draw` to `Raterizer::Draw` in the tracing events (CC @filiph ). I think it's fine to leave `kRasterizeFrameEventName` as is (since the change needs to start with the engine repo), and just remove `GPU` from this documentation. The `engine rasterizer` would be more accurate than `GPU rasterizer` because there's no longer `GPURasterizer` class in the engine. There's only `Rasterizer`.

Reply:
Didn't we intentionally keep names of timeline events so as to not break people's measurement scripts?

Reply:
Yes, now I remembered! You and Nathan were explicitly discussing about avoiding changing `GPURasterizer` in the tracing. Now it's also clear that if we change the `GPURasterizer` label in the engine, it will also break our framework performance tests.